ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/DdlDmlRequestValidators.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client.impl;
+
+import io.confluent.ksql.api.client.exception.KsqlClientException;
+import java.util.concurrent.CompletableFuture;
+
+final class DdlDmlRequestValidators {
```

Comment:
Not a big deal but is it really worth putting this method in its own class? Might be simpler to just have the method in the class where it's used.

Reply:
Agreed this is a bit overkill but I think it's better than having the logic in ClientImpl (where the method is needed) as ClientImpl is already pretty massive. Will leave this for now.